ci-operator/step-registry/gather/gcp-console/gather-gcp-console-commands.sh

```diff
@@ -6,8 +6,14 @@ set -o pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
-export GOOGLE_CLOUD_KEYFILE_JSON="${CLUSTER_PROFILE_DIR}/gce.json"
-gcloud auth activate-service-account --key-file="${GOOGLE_CLOUD_KEYFILE_JSON}"
+if [ -f "${SHARED_DIR}/gcp_c3_metal_sa.json" ]; then
```

Comment:
I guess we do not need the condition any more after we created the dedicated cluster profile.

Reply:
Yes, you are right. Let me update it.